lib/mp4/transmuxer.js

```diff
@@ -1180,5 +1202,5 @@ module.exports = {
   AUDIO_PROPERTIES: AUDIO_PROPERTIES,
   VIDEO_PROPERTIES: VIDEO_PROPERTIES,
   // exported for testing
-  generateVideoSegmentTimingInfo: generateVideoSegmentTimingInfo
+  generateSegmentTimingInfo: generateSegmentTimingInfo
```

Comment:
Technically we only exported it for testing. I highly doubt anyone is using it. Though maybe it's best to remove it in the main build.

Reply:
Looking at it more closely, it seems like we're fine. We only re-export the Transmuxer and Video and Audio segment streams from the transmuxer module rather than the object returned by the module.